specification/consumption/resource-manager/Microsoft.Consumption/stable/2018-01-31/examples/MarketplacesListForBillingPeriod.json

```diff
@@ -0,0 +1,43 @@
+{
+  "parameters": {
+    "api-version": "2018-01-31",
+    "scope": "subscriptions/subid/providers/Microsoft.Billing/billingPeriods/201702"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "value": [
+          {
+            "id": "/subscriptions/subid/providers/Microsoft.Billing/billingPeriods/201702/providers/Microsoft.Consumption/marketplaces/marketplacesId1",
+            "name": "marketplacesId1",
+            "type": "Microsoft.Consumption/marketplaces",
+            "tags": {
+              "env": "newcrp",
+              "dev": "tools"
+            },
+            "properties": {
+              "billingPeriodId": "/subscriptions/subid/providers/Microsoft.Billing/billingPeriods/201702",
+              "usageStart": "2017-02-13T00:00:00Z",
+              "usageEnd": "2017-02-13T23:59:59Z",
+              "instanceName": "shared1",
+              "instanceId": "/subscriptions/subid/resourceGroups/Default-Web-eastasia/providers/Microsoft.Web/sites/shared1",
+              "currency": "USD",
+              "consumedQuantity": 0.00328,
+              "pretaxCost": 0.67,
+              "isEstimated": false,
+              "meterId": "fe167397-a38d-43c3-9bb3-8e2907e56a41",
+              "offerName": "offer1",
```

Comment:
It is `offerType`

Reply:
@mcardosos renamed it. Should be offerName, changed in mode.